.github/workflows/mssql.yml

```diff
@@ -74,14 +73,11 @@ jobs:
           ini-values: date.timezone='UTC'
           coverage: pcov
 
-      - name: Update composer.
-        run: composer self-update
+      - name: Install Composer dependencies
+        uses: ramsey/composer-install@v3
 
-      - name: Install db and db-mssql.
-        uses: yiisoft/actions/install-packages@master
-        with:
-          packages: >-
-            ['db', 'db-mssql']
```

Comment:
The same for other tests

Reply:
This leads to commands:

```shell
composer require "yiisoft/db:dev-master as 2.0.0"
composer require "yiisoft/db-sqlite:dev-master as 2.0.0"
```

`dev-master` is not right. In actual usage will be installed `^2.0`, no `dev-master`.

Reply:
Yes, but this is required if changes in `db` also required for `db-migration`. The same as for `db-pgsql` it also requires `^2.0` but when you develop next minor or major it also required `dev-master` or the same branch name.

Reply:
It can be several PRs one by one:
 - if PR requires changes in `db` we use the same branch name
 - if the next PR does not require changes in `db` we use `dev-master` which has changes made before.